app/views/components/_header.html.erb

```diff
@@ -1,13 +1,13 @@
 <header class="sdn-header sdn-header-950" role="banner" data-module="sdn-header">
   <div class="sdn-header__container govuk-width-container">
     <%= link_to root_path, class: 'sdn-header__link' do %>
-      <span class="sdn-header__product-name">Návody.Digital</span>
+      <%= image_tag 'logo.svg' %>
```

Comment:
Toto je doladene? Viem, ze tam bol problem s vertikalnym centrovanim menu.

Reply:
Riešili to už v inom PR. revert